libs/@blockprotocol/graph/src/stdlib/subgraph/element/entity.ts

```diff
@@ -54,80 +66,94 @@ export const getEntity = (
 
   const revisionVersions = Object.keys(entityRevisions).sort();
 
-  let entityRevision: Entity | undefined;
-
   // Short circuit for efficiency, just take the latest
   if (targetRevisionInformation === undefined) {
-    [entityRevision] = revisionVersions
+    return revisionVersions
```

Comment:
Doubt it makes any performance difference but maybe marginally clearer
```ts
    const lastIndex = revisionVersions.length - 1;
    return entityRevisions[revisionVersions[lastIndex]!]!.inner;
```

Reply:
Yeah fair enough, I just hate being forced into shoving `!`'s in places where iterator functions can convince the compiler it's okay

Reply:
[Replace slice(-1) syntax](https://github.com/blockprotocol/blockprotocol/pull/928/commits/538ac50b82e2f9d62a01b9c9383f70e2536d0cc2)